### DISCUSSION TITLE
Pages plugin sort by name

### DISCUSSION THREAD

**Discussion by hibangmike** [wintercms/winter#104] [Ideas]
Hi,

I don't know where to post this. But regarding the static pages plugin, can I make a small request for sorting the layout in dropdown by filename ? Currently I see it is based on creation date. So I think it would be better, especially for end users who operate the site to have sort by filename.

Well maybe it can extend beyond the static pages plugin and in the CMS editor also can make the layout sort order by filename ?

Thanks

**Comment by bennothommo** [comment]
Thanks for the suggestion. I'll change this to an "Ideas" thread as it is more a request for a new feature as opposed to a question.

**Comment by LukeTowers** [comment]
@hibangmike can you link me the code where it's sorted by creation date? I'd agree that it should be sorted by filename too, in fact that's what I thought it was already sorted as.
